src/conditional.fs

```diff
@@ -330,7 +333,7 @@ let pre_synth (rho:formula, econd:formula, vars) =
 /// with phase change. 
 let rec pre_synth_phase (rho:formula, vars, cur_constr:formula, econd, depth_left) = 
 
-
+    printfn "HELLO!"
```

Comment:
I can start coughing too, though - are there any other changes I made to
files different form conditional.fs that you didn't commit, apart of the
one in formula.fs?

On 7 October 2015 at 14:47, Maria Gorinova <m.gorinova@gmail.com> wrote:

> Sorry, that's something automatically generated after me trying to run T2
> for ages. I probably don't need it. Sorry, I'm doing three things at the
> same time and I'm kind of overlooking all of them. Let mi fix this.
>
>
>
> On 7 October 2015 at 14:38, Marc Brockschmidt <notifications@github.com>
> wrote:
>
>> In src/conditional.fs
>> <https://github.com/mmjb/T2/pull/11#discussion_r41390475>:
>>
>> > @@ -330,7 +333,7 @@ let pre_synth (rho:formula, econd:formula, vars) =
>> >  /// with phase change.
>> >  let rec pre_synth_phase (rho:formula, vars, cur_constr:formula, econd, depth_left) =
>> >
>> > -
>> > +    printfn "HELLO!"
>>
>> *cough*
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/mmjb/T2/pull/11/files#r41390475>.
>>
>
>